packages/e2e/test/artillery/wallet-restoration/WalletRestoration.ts

```diff
@@ -64,15 +64,6 @@ const extractAddresses = async (count: number): Promise<AddressesModel[]> => {
   return result;
 };
 
-const mapToGroupedAddress = (addrModel: AddressesModel): GroupedAddress => ({
```

Comment:
The extraction of this util should be done first in a separate commit, of type `test`.

Reply:
Done